desktop/src/main/java/bisq/desktop/main/account/content/fiataccounts/FiatAccountsDataModel.java

```diff
@@ -124,6 +125,14 @@ public void onSaveNewAccount(PaymentAccount paymentAccount) {
         accountAgeWitnessService.signAndPublishSameNameAccounts();
     }
 
+    public void onUpdateAccount(PaymentAccount paymentAccount) {
+        user.requestPersistence();
+        SepaAccount account = (SepaAccount) paymentAccount;
+        account.getAcceptedCountryCodes();
+        log.info("ON UPDATE, ACCOUNT: " + account.getAcceptedCountryCodes());
+        // TODO: persist changes
```

Comment:
Is this PR already ready for review?

Reply:
Sorry, now it's okay. This TODO was just leftover.